lib/msal-browser/src/controllers/IController.ts

```diff
@@ -3,6 +3,78 @@
  * Licensed under the MIT License.
  */
 
-import { IPublicClientApplication } from "../app/IPublicClientApplication";
+import {
+    AuthenticationResult,
+    AccountInfo,
+    Logger,
+    PerformanceCallbackFunction,
+    IPerformanceClient,
+    ICrypto,
+    CommonSilentFlowRequest
+} from "@azure/msal-common";
+import { RedirectRequest } from "../request/RedirectRequest";
+import { PopupRequest } from "../request/PopupRequest";
+import { SilentRequest } from "../request/SilentRequest";
+import { SsoSilentRequest } from "../request/SsoSilentRequest";
+import { EndSessionRequest } from "../request/EndSessionRequest";
+import { ApiId, InteractionType, WrapperSKU } from "../utils/BrowserConstants";
+import { INavigationClient } from "../navigation/INavigationClient";
+import { EndSessionPopupRequest } from "../request/EndSessionPopupRequest";
+import { ITokenCache } from "../cache/ITokenCache";
+import { AuthorizationCodeRequest } from "../request/AuthorizationCodeRequest";
+import { BrowserConfiguration } from "../config/Configuration";
+import { BrowserCacheManager } from "../cache/BrowserCacheManager";
+import { NativeMessageHandler } from "../broker/nativeBroker/NativeMessageHandler";
+import { EventHandler } from "../event/EventHandler";
+import { PopupClient } from "../interaction_client/PopupClient";
+import { SilentIframeClient } from "../interaction_client/SilentIframeClient";
 
-export type IController = IPublicClientApplication;
+export interface IController {
```

Comment:
nit: Can we please have spaces between APIs for readability? What do we do today across headers in our library?

Reply:
done.